### PR TITLE
Add Allnodes RPC endpoints to various blockchains

### PR DESCRIPTION
**Pull request by 3eph1r0th**

### CODE REVIEW

File: cudos/chain.json [hunk 1]
```diff
@@ -164,6 +168,10 @@
       {
         "address": "https://cudos-api.lavenderfive.com/",
         "provider": "Lavender.Five Nodes 🐝"
+      },
+      {
+        "address": "https://celestia-rest.publicnode.com",
```
Comment (JeremyParish69) [inline cudos/chain.json:173]: this says celestia but it's in cudos' chain.json

Comment (3eph1r0th) [reply cudos/chain.json]: Thank you. Fixed

**Comment by reversesigh** [comment]
@3eph1r0th resolve conflicts and then we can merge

**Comment by 3eph1r0th** [comment]
@reversesigh should be fixed now

**Pull request merged**